mixer/testdata/config/signalfx.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: "config.istio.io/v1alpha2"
```

Comment:
@geeknoid should we put these test configs in the adapter directory instead?  By having all of these configs in `mixer/testdata/config`, starting a local Mixer now ends up starting every single adapter -- with a lot of failures because associated backends are not present in local envs, etc.

Reply:
Yeah, it'd be better to have those local, assuming things build & run correctly that way.

Reply:
Ok, I'll follow the `operatorconfig/` dir pattern some of the other adapters use.